testsuite/features/step_definitions/command_steps.rb

```diff
@@ -56,7 +56,8 @@
     # TODO: move that internal resource to some other external location
     STDERR.puts 'Sanity check not implemented, move resource to external network first'
   else
-    url = 'https://portus.mgr.suse.de:5000'
+    portus_uri = ENV['PORTUS_URI']
```

Comment:
See my remarks on sumaform PR: an alternative would be to make this server non-mandatory. In that case, that variable could be empty, and those whole tests would be skipped.

Reply:
I made both registries now optional. I added tags to exclude the tests.